src/fetchtastic/download/client_app.py

```
@@ -0,0 +1,1157 @@
+    )
+
+
+class MeshtasticClientAppDownloader(BaseDownloader):
+    """
+    Downloader for Meshtastic client app assets.
+
+    This is the primary lifecycle owner for app/<version>/ and
+    app/prerelease/<version>/.
+    """
+
+    def __init__(self, config: dict[str, Any], cache_manager: CacheManager):
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)